src/inference/src/dev/core_impl.cpp

```diff
@@ -1436,9 +1436,15 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model_and_cache(const std::s
         try {
             // need to export network for further import from "cache"
             OV_ITT_SCOPE(FIRST_INFERENCE, ov::itt::domains::LoadTime, "Core::compile_model::Export");
+            std::string compiled_model_format("");
```

Comment:
nitpick: by default (with default constructor) std::string is initialized with empty string
```suggestion
            std::string compiled_model_format;
```